src/main/kotlin/org/rust/cargo/project/workspace/RsAdditionalLibraryRootsProvider.kt

```diff
@@ -25,6 +25,9 @@ class CargoLibrary(
     private val isStd: Boolean
 ) : SyntheticLibrary(), ItemPresentation {
     override fun getSourceRoots(): Collection<VirtualFile> = listOf(root)
+    override fun getExcludedRoots(): Set<VirtualFile> =
+        if (isStd) root.findChild("tests")?.let { setOf(it) } ?: emptySet() else emptySet()
```

Comment:
Some std crates contain also `benches` folder with benchmarks. Looks like that we should exclude them as well

Reply:
Also, it would be great to mark out a content of each stdlib crate manually and pass such info into a package
But I think it's ok for now